cypress/support/commands.ts

```diff
@@ -67,10 +56,23 @@ Cypress.Commands.add("getInputByLabel", (label) =>
     })
 );
 
-Cypress.Commands.add("toggleTableFilter", (colNum) => {
+Cypress.Commands.add("toggleTableFilter", (colNum: number) => {
   cy.get(`.ant-table-thead > tr > :nth-child(${colNum})`)
     .find("[role=button]")
     .first()
     .click();
   cy.get(".ant-table-filter-dropdown").should("be.visible");
 });
+
+Cypress.Commands.add(
+  "paste",
+  { prevSubject: true },
+  (selector: Element, pastePayload: string) => {
+    // We can paste directly into a textarea but the onchange event is not fired because cypress directly manipulates the dom elements value.
+    // So we need to manually type in the last character of the paste payload to trigger an onchange event.
+    const subString = pastePayload.substr(0, pastePayload.length - 1);
+    const lastChar = pastePayload.slice(-1);
+    cy.wrap(selector).invoke("val", subString);
```

Comment:
I think we can get a better representation of copy and paste if we actually write to a clipboard and then paste. Cypress shows us an example of this here: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/testing-dom__clipboard/cypress/integration/spec.js#L79

Reply:
I tried this it doesn't seem to work in chrome. 
That file states that this method only works in electron. 
> // access to the clipboard reliably works in Electron browser
// in other browsers, there are popups asking for permission
// thus we should only run these tests in Electron